pkg/detectors/aha/aha_test.go

```diff
@@ -108,6 +108,8 @@ func BenchmarkFromData(benchmark *testing.B) {
 	s := Scanner{}
 	for name, data := range detectors.MustGetBenchmarkData() {
 		benchmark.Run(name, func(b *testing.B) {
+			b.ResetTimer()
+			b.ResetTimer()
```

Comment:
Double reset here

Reply:
ah whoops thanks, yea i was probably testing with this one and it got added with the bulk find and replace.